.github/workflows/npm-release-publish.yml

```diff
@@ -25,6 +25,14 @@ concurrency:
   cancel-in-progress: true
 
 jobs:
+  print-inputs:
+    runs-on: ubuntu-latest
+    steps:
+      - name: job summary
+        run: |
+          echo "### inputs" >> $GITHUB_STEP_SUMMARY
```

Comment:
Can we also output the branch this is run on?
![image](https://user-images.githubusercontent.com/2684369/204922464-14546bbf-0a2e-4be2-b282-7a47ad392942.png)